web/res/js/membershipForm.js

```diff
@@ -150,12 +150,8 @@ $( function () {
 				stateKey: 'membershipFormContent.membershipType'
 			},
 			{
-				viewHandler: WMDE.View.createFeeOptionSwitcher( $( '#amount-1' ), 1 ),
-				stateKey: 'membershipFormContent.paymentIntervalInMonths'
-			},
-			{
-				viewHandler: WMDE.View.createFeeOptionSwitcher( $( '#amount-2' ), 6 ),
-				stateKey: 'membershipFormContent.paymentIntervalInMonths'
+				viewHandler: WMDE.View.createFeeOptionSwitcher( [ $( '#amount-1' ), $( '#amount-2' ), $( '#amount-3' ), $( '#amount-4' ), $( '#amount-5' ), $( '#amount-6' ), $( '#amount-7' ) ], { person: 24, firma: 100 } ),
+				stateKey: 'membershipFormContent'
```

Comment:
So the amounts are hardcoded here... If I'm not mistaken in the PHP they come from the config. Is there a way in the JS to get those values yet?